src/Umbraco.PublishedCache.NuCache/DataSource/BTree.cs

```diff
@@ -22,6 +22,13 @@ internal class BTree
                 // default is 4096, min 2^9 = 512, max 2^16 = 64K
                 FileBlockSize = GetBlockSize(settings),
 
+
+                //HACK: Forces FileOptions to be WriteThrough here: https://github.com/mamift/CSharpTest.Net.Collections/blob/9f93733b3af7ee0e2de353e822ff54d908209b0b/src/CSharpTest.Net.Collections/IO/TransactedCompoundFile.cs#L316-L327,
+                // as the reflection uses otherwise will failed in .NET Core as the "_handle" field in FileStream is renamed to "_fileHandle".
+                StoragePerformance = StoragePerformance.CommitToDisk,
```

Comment:
I dont have a lot of knowledge about the b-tree package, so I don't know whether `StoragePerformance.CommitToDisk,` is a bad thing, but I will make a PR to the project, about changing the `_handle` variable

Reply:
Sounds good but do note that the orig maintainer of this lib prob won't release anything but he may respond. Stephane had been in touch with him in the past and he was responsive but said he isn't actually 'maintaining' the project. He said it was used in hundreds+ different projects and it's stable, however with the netcore change this might of course be different. Would be interesting to see what they know about this one.

Reply:
Right now we use a fork of the original project. That fork is hopefully more active